### DISCUSSION TITLE
Menampilkan logo di shiny

### DISCUSSION THREAD

**Discussion by myibra** [teamalgoritma/community#94] [Q&A] [answered]
Saya ingin menampilkan logo di bagian 'title' dashboard berikut code yang saya sudah coba : 

>title <- tags$a(tags$img(src="logo.jpg", height = '50', width = '50'))
>navbarPage(title = title)

berikut hasil nya:

<img width="450" alt="image" src="https://user-images.githubusercontent.com/50072768/167776512-becc1075-b3ab-4f7b-a04b-d3e640787583.png">

sekiranya tidak muncul gambar tersebut karena apa ya?
terimakasih

**Comment by VictorNugraha** [comment, marked as answer]
Halo, @myibra!

Apakah Bapak sudah meletakan logo yang ingin digunakan ke sebuah folder yang diberikan nama **_www_**? Jika belum, boleh tolong dibuat sebuah folder dengan nama **_www_**, penamaan harus  _**www**_ dengan tujuan untuk menyamarkan foto yang ingin dilampirkan dari local sebagai sebuah URL (sejauh ini ketika kita ingin melampirkan gambar ataupun logo dari local host ke server Shiny, kita harus menyertakannya dalam bentuk URL). 

NB: Ketika Bapak ingin langsung merender logo atau gambar dari langsung dari ui.R, Bapak tidak perlu mengspesifikan directory _**www**_ (seperti yang akan Bapak lakukan untuk memunculkan logo). Akan tetapi jika Bapak, menggunakan fungsi renderImage() pada bagian server, Bapak harus mengspesifikan directory _**www**_.

Demikian informasi yang dapat saya sampaikan, semoga menjawab pertanyaan yang diberikan. Selamat meconba :D

**Reply by myibra** [reply]
thankyou mas victor, sudah berhasil ketika dimasukan kedalama folder www

**Reply by myibra** [reply]
Halo mas Victor, 

Mau bertanya terkait menampilkan image di shinydashboard.  Jadi saya ingin membuat footer dan melihat beberapa referensi pada menggunakan html. berikut adalah code yang saya coba buat untuk footer.html:

```
<hr>

<div class="container">
  <img src="images/logo.png" alt="logo" class="center-block img-responsive" width="50" style="margin-bottom: 80px;" />

  <!-- FOOTER -->
  <footer>
  <p class="pull-right"><a href="#">Back to top</a></p>
  <p>Algoritma, Data Visualization Capstone Project&mdash;Rhea Night</p>
  </footer>

</div>
```

dan berikut adalah hasilnya :

<img width="1277" alt="image" src="https://user-images.githubusercontent.com/50072768/168210712-8bd6d661-8a27-4e7a-bc74-82d40a8d8366.png">

dimana hasilnya hanya muncul untuk tulisan saja dan gambar logo nya tidak berhasil muncul. Untuk path `"images/logo.png"` sudah di dalam folder `www` . Sekiranya kurang di bagian mana ya mas?

**Reply by VictorNugraha** [reply]
Halo, @myibra 

Hal tersebut mungkin saja terjadi karena peletakan code  `<div class="container">
  <img src="images/logo.png" alt="logo" class="center-block img-responsive" width="50" style="margin-bottom: 80px;" />`, di luar `<footer> </footer>`. Bisa dicoba untuk memasukan code tersebut di antara `<footer> </footer>`.

Demikian informasi yang dapat kami samapaikan, semoga menjawab pertanyaan yang diberikan. Selamat mencoba :)
